### PR TITLE
Test kit setup

### PR DESCRIPTION
**Pull request by elsaperelli**
# Summary
This pull request sets up a test kit for the measure repository service using the [inferno-template](https://github.com/inferno-framework/inferno-template). Files that were not from the inferno-template are modeled after our other test-kit: [deqm-test-kit](https://github.com/projecttacoma/deqm-test-kit).

## New behavior
- Everything! Brand new repository woohoo.
- There are no measure repository service specific tests yet, just ones that came with the inferno-template to show that the test kit can be run. 
- In addition to stuff that was in the inferno-template, I added rubocop as a dependency and added Github workflows and a PR template.

## Code changes
- `.github/pull_request_template.md` - classic PR template.
- `github/workflows/ruby.yml` - I pulled this file from the `deqm-test-kit`.
- `Dockerfile` - deleted comments from the inferno-template.
- `Gemfile.lock` - added rubocop as a dependency and changed inferno_template to measure_repository_service_test_kit.
- `docker-compose.background.yml` - changed inferno_template to measure_repository_service_test_kit.
- `inferno_template.gemspec` is now `measure_repository_service_test_kit.gemspec`.
- `lib` directory now contains a `measure_repository_service_test_kit` directory that contains the `patient_group.rb` file (from the inferno-template) and an `igs` directory that contains `gm_stu3_ig.tgz` which is the `.tgz` file for the entire [Quality Measure IG](http://hl7.org/fhir/us/cqfmeasures/STU3/). It also contains a `measure_repository_service_test_kit.rb` file.
- `setup.sh` / `run.sh` - I took the "-" out of `docker-compose` in these files because it wasn't working for Nat. Both of them worked for me.
- `spec/inferno_template` directory was renamed to `spec/measure_repository_service_test_kit`.

# Testing guidance
- Take a look at the [inferno-template](https://github.com/inferno-framework/inferno-template) to see how I got everything set up. 
- Look through all the files changed in this PR but also look at the repository as a whole since I just set it up from scratch. Let me know if I am missing anything!
- To start up the test-kit (remember, there are no measure-repository-service specific tests yet), run `npm start` in the `measure-repository-service` repository and run `./setup.sh` in another tab your terminal in the `measure-repository-service-test-kit`.
- When `./setup.sh` is complete, run `./run.sh` in the `measure-repository-service-test-kit` repository and navigate to `http://localhost` to access the test suite!

### CODE REVIEW

**Review by natjoe4 — COMMENTED** [review]
Great work! I think the patient_group.rb file and patient_group_spec files can be deleted, unless we want to keep them as references for when we write actual test files. Otherwise, no comments!!

File: lib/measure_repository_service_test_kit/patient_group.rb [hunk 1]
```diff
@@ -1,4 +1,7 @@
-module InfernoTemplate
+# frozen_string_literal: true
+
+module MeasureRepositoryServiceTestKit
+  # Example Patient group test suite
```
Comment (mgramigna) [inline lib/measure_repository_service_test_kit/patient_group.rb:4]: we should probably just delete this right? It's going to get immediately deleted anyway when we do our first task. It's fine to just leave the capability statement test in and get rid of the patient group

**Review by mgramigna — CHANGES_REQUESTED** [review]

File: lib/measure_repository_service_test_kit/patient_group.rb (hunk 1 above)
Comment (elsaperelli) [reply lib/measure_repository_service_test_kit/patient_group.rb:4]: Sounds good! I wasn't sure if we wanted to keep an example or anything

File: docker-compose.yml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-version: '3'
+version: "3"
```
[the remark this answers is not in the document]
Comment (mgramigna) [reply docker-compose.yml:1]: Just re-read the instructions and realized it's hosted on just `localhost` haha. I think adding the ports makes it a little clearer

**Review by natjoe4 — APPROVED** [review]
Looking good!

**Review by mgramigna — APPROVED** [review]

**Pull request merged**